README.md

```diff
@@ -47,6 +47,16 @@ The <a href="https://layer5.io/performance">Service Mesh Benchmark Specification
 <br /><br /><br />
 </p>
 
+<p style="clear:both;">
+<h2><a href="https://layer5.io/projects/image-hub">Image Hub</a></h2>
+
+<a href="https://layer5.io/projects/image-hub"><img src="assets/images/image-hub/layer5-image-hub.png"
+style="float:left;margin:10px;" width="125px"
+alt="Image Hub" align="left" /></a>
+<a href="https://layer5.io/projects/image-hub">Image Hub</a> is a demo application which runs on <a href="https://meshery.layer5.io/docs/service-meshes/adapters/consul">Console</a> and facilitates exploring WebAssembly modules used as Envoy Filters.
```

Comment:
Consul

Reply:
Good eye.

Thank you for this, @chandrashritii 👍

Reply:
@kushthedude Thanks for pointing that out! Fixed the typo.

@leecalcote Happy to contribute :) Hope it can be merged now